src/components/Header.spec.js

```diff
@@ -0,0 +1,10 @@
+import {render, screen} from '@testing-library/react'
+  import Header from './Header'
+    
+  describe('Header', () => {
+      it('renders', () => {
+          render(<Header />)
+          expect(screen.getByText('Header')).toBeInTheDocument()
```

Comment:
This will fail I guess, since you are not giving the Header component the text "Header" to render as a child. Since getByText searches for text inside of a component while not concerning about the tag/component-names you won't  get the result you want out of it. 
```suggestion
  describe('Header', () => {
      it('renders', () => {
          render(<Header >testText<Header>)
          expect(screen.getByText('testText')).toBeInTheDocument()
```